cristin-import/src/main/java/no/unit/nva/cristin/lambda/CristinRerunErrorsEventEmitter.java

```diff
@@ -0,0 +1,110 @@
+package no.unit.nva.cristin.lambda;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.events.models.EventReference;
+import no.unit.nva.publication.s3imports.PutSqsMessageResult;
+import no.unit.nva.publication.s3imports.SqsBatchMessenger;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.ioutils.IoUtils;
+import nva.commons.core.paths.UnixPath;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class CristinRerunErrorsEventEmitter implements RequestStreamHandler {
+
+    public static final String INPUT_FIELD = "input";
+    public static final String FILE_URI_FIELD = "fileUri";
+    public static final String DATA_IMPORT_TOPIC = "PublicationService.DataImport.DataEntry";
+    public static final String CRISTIN_DATA_ENTRY_SUBTOPIC = "PublicationService.CristinData.DataEntry";
+    public static final Environment ENVIRONMENT = new Environment();
+    public static final String SQS_BATCH_RESULT_MESSAGE = "Sqs batch result: {}";
+    public static final String REPORTS_DELETED_MESSAGE = "Old reports have been deleted!";
+    private static final Logger logger = LoggerFactory.getLogger(CristinRerunErrorsEventEmitter.class);
+    private static final String CRISTIN_IMPORT_BUCKET = ENVIRONMENT.readEnv("CRISTIN_IMPORT_BUCKET");
+    private static final String CRISTIN_ENTRY_QUEUE = ENVIRONMENT.readEnv("CRISTIN_IMPORT_DATA_ENTRY_QUEUE_URL");
+    private final S3Driver s3Driver;
+    private final SqsBatchMessenger batchMessenger;
+
+    @JacocoGenerated
+    public CristinRerunErrorsEventEmitter() {
+        this.s3Driver = new S3Driver(S3Driver.defaultS3Client().build(), CRISTIN_IMPORT_BUCKET);
+        this.batchMessenger = new SqsBatchMessenger(defaultAmazonSQS(), CRISTIN_ENTRY_QUEUE);
```

Comment:
use this(new S3Driver(S3Driver.defaultS3Client().build(), CRISTIN_IMPORT_BUCKET), new SqsBatchMessenger(defaultAmazonSQS(), CRISTIN_ENTRY_QUEUE)) instead, that way the default constructor stays in sync with the parameterised constructor.